src/Promitor.Core.Scraping/Configuration/Serialization/v1/Providers/TrafficManagerProfileDeserializer.cs

```diff
@@ -0,0 +1,14 @@
+using Microsoft.Extensions.Logging;
+using Promitor.Core.Scraping.Configuration.Serialization.v1.Model.ResourceTypes;
+
+namespace Promitor.Core.Scraping.Configuration.Serialization.v1.Providers
+{
+    public class TrafficManagerProfileDeserializer : ResourceDeserializer<TrafficManagerProfileResourceV1>
+    {
+        public TrafficManagerProfileDeserializer(ILogger<TrafficManagerProfileDeserializer> logger) : base(logger)
+        {
+            Map(resource => resource.ProfileName)
```

Comment:
I'm not sure which one to use, I think "Name" is simpler for people to understand what is required as "profile name" is not really shown in portal for example
```suggestion
            Map(resource => resource.Name)
```